src/enums/variant-payloads.md

```diff
@@ -9,8 +9,15 @@ You can define richer enums where the variants carry data. You can then use the
 
 <details>
 
+Key Points:
+  
+* This might be the first time seeing pattern matching so perhaps introduce the topic, more details are provided in further slides.
+  * A match expression can return any type.
```

Comment:
Using the word "return" here is dangerous: returning has a specific meaning in a programming language and the `match` expression doesn't actually return anything.

Instead, please say something like "The `match` expression has a value. The value is the last expression in the match arm which was executed".